README.md

```diff
@@ -1,24 +1,58 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Recipe App
 
-Things you may want to cover:
+> This projects was built with ruby
 
-* Ruby version
+Additional description about the project and its features.
```

Comment:
- [ ] Add a clear project description to the readme file. Remember this is part of what constitutes a professional readme file.